src/Response.php

```diff
@@ -29,7 +29,7 @@ public function getData()
         $data = parent::getData();
         if ($data === null) {
             $content = $this->getContent();
-            if (!empty($content)) {
+            if (is_string($content) && strlen($content) > 0) {
```

Comment:
```suggestion
            if ($content !== '') {
```

Reply:
- `$content` is `null` when not set
- `curl_exec` and `stream_get_contents` will return boolean values on error. Do they throw exceptions when error? Will `$content` be a `false`?

Will upper conditions case parse errors?

Reply:
Is there anything I can do to push it forward? Or just commit as the suggestion?

Reply:
On second thought, I would ignore this suggestion. For me both these scenarios could work incorrectly, but I would not touch them for the sake of BC and to no stuck with this PR forever.

Reply:
Thank you for quick responding. It's my first time contributing. And I don't know what to do next. Could you give me some advice on this PR / Issue #224 ?

Reply:
I think you just need to wait for someone from Yii team to review end merge this PR, there nothing more to do IMO.

Reply:
:sob: All right. Thank you! I will use a wrapper for the requests to solve the problem in my project.